.evergreen/start-atlas-cloud-cluster.sh

```diff
@@ -89,6 +89,9 @@ cleanup() {
 
 trap cleanup EXIT
 
+echo "Logging in to docker"
+echo "${DOCKERHUB_PASSWORD}" | docker login -u ${DOCKERHUB_USERNAME} --password-stdin
```

Comment:
Not to block this, but this script is also supposed to be used locally to run these tests, what will happen if these are missing? Is it just silently fails? Does it mess with the login that docker desktop is requiring on macos?